### ISSUE TITLE
v2.4.8-p8 introduces breaking change

### ISSUE THREAD

**Issue by troublete** [tubalmartin/YUI-CSS-compressor-PHP-port#37]
Hello,
i'm working on a large enterprise application. With the patch version `p8` you introduced a calculated value (https://github.com/tubalmartin/YUI-CSS-compressor-PHP-port/blob/master/cssmin.php#L35) that is a breaking change because it requires >=PHP 5.6.

For us this is a crucial issue. We rolled back to `p7` for now, but it is a crucial misinterpretation of SemVer. 

Please consider a invalidating of the patch version and release a new major. 

Best regards. :)

**Comment by tubalmartin** [comment]
Oopssorry for that. My mistake...I will release p9 right away rolling back those changes.

**Comment by troublete** [comment]
Hey, please don't do that! 
It would be a redundant solution. Because technically it is possible to use `p7`. What the real solution would be is to create a `v3`. That will update the composer required php version to 5.6, since it is a new major change. And invalidate the `p8`.

**Comment by tubalmartin** [comment]
Ummm yeah, but I want to support php 5.0

**Comment by troublete** [comment]
Yeah but you do, on your package version `2.8`. So every package that now requires your your package von `2.8` will still get the latest addition.

That's why i'm saying. Invalidate the `p8` -- take the changes and release a `v3` that has php 5.6 as minimum requirement and you're fine.

Also you only should use postfixed versions if you really have to. Also there is a fine distinction between a `+` and a `-` patch. So technically you would need to release patch versions with `+` since they are an addition to the current minor.

**Comment by futtta** [comment]
Going for php>=5.6 would break Autoptimize (which uses YUI-CSS-compressor-PHP-port) for a considerable subset of users of my WordPress plugin who are still on older versions of PHP. so +1 for a rollback :-)

**Comment by tubalmartin** [comment]
Thank you for reporting the issue @troublete . I understand your concern about releasing a v3 but IMHO it would only make sense to release that one if I were to support only PHP 5.6 onwards but, as I said, that's not my intent since there's no need for it, that's way I've just released patch 9 rolling back those changes to support PHP 5.0.
